specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/blob.json

```diff
@@ -797,6 +800,14 @@
   "definitions": {
     "ContainerProperties": {
       "properties": {
+        "defaultEncryptionScope": {
```

Comment:
Adding new writable properties to a stable API is considered a breaking change. Could this instead be moved into a new API version?

Reply:
Add these 2 new properties should no be breaking since:
1. Old sdk still work, as the new properties are optional
2. Old SDK won't change the properties added with new SDK